.env.sample

```diff
@@ -0,0 +1,4 @@
+TEST_PK_1='put your private key here'
+TEST_Pk_2='put your private key here'
+TEST_Pk_3='put your private key here'
+TEST_SEED_1='pablo diego jose francisco de paula juan nepomuceno maria ruiz y picasso'
```

Comment:
It's enough with `TEST_SEED` as name. It will be just one

Reply:
Okay!